ui/lib/kv/addon/components/kv-list-filter.js

```diff
@@ -29,8 +29,18 @@ import { tracked } from '@glimmer/tracking';
 
 export default class KvListFilterComponent extends Component {
   @service router;
+  @tracked query;
   @tracked filterIsFocused = false;
 
+  constructor() {
+    super(...arguments);
+    this.query = this.args.filterValue;
```

Comment:
Is `filterValue` always passed into this component?

Reply:
Typically yes, it's the query param for the page. So we're basically using it here to set query to the initial value (which is `@filterValue`). To address your other question, they differ once the user types more into the input, but I think we could remove that getter since we are no longer disabling the input when the query hasn't been changed